nf_wochenende.nf

```diff
@@ -479,7 +482,8 @@ process heattrees {
 process heatmaps {
     cpus = 1
 
-    conda params.conda_haybaler
```

Comment:
Why remove the conda environment, is it not useful for heattrees?

Reply:
According to the Haybaler Readme, conda needs to be deactivated for heatmaps: 
https://github.com/MHH-RCUG/haybaler#heatmaps  
```# go to the haybaler output file
# Now run the Rscript to create a heatmap, this requires an R installation.
# Because of a bug with heatmaply and conda, no conda enviroment is allowed to be activated
conda deactivate
bash runbatch_heatmaps.sh 
```

Reply:
true. Then the heatmaps are all run in  a local R installation. I'll have to sort one out and also add parameters to do the visualization stages separately to the main Wochenende and other stages ..... or just start manually.

I wanted to make each optional stage individually runnable anyway by parameter without having to comment anything out in the `.nf ` file.